### ISSUE TITLE
Docs: Community - Need help with a Vega/Vega-Lite specification?

### ISSUE THREAD

**Issue by Giammaria** [deneb-viz/deneb#414]
I have an idea for a new section in the community area of the documentation and am curious to find out if others would find this beneficial. There are a lot of great resources listed currently in that section of the documentation, but I'm thinking adding a section for specifically requesting help from others in the community with Vega/Vega-Lite specifications could be helpful to others. I know you're plenty busy with many spinning plates, @dm-p, so this could be one less plate. I'm thinking that the individual's forum of choice for providing assistance will also be important.

The following is off the cuff, but this is what I had in mind:
***
### **Need help with a Vega/Vega-Lite specification?**
The following individuals are willing to provide assistance. Reach out to them on their preferred forum/method of contect.

- **Power BI Forums** 
  - Madison Giammaria - [giammariam](https://community.fabric.microsoft.com/t5/user/viewprofilepage/user-id/124852)
  - Name 2 - username
  - Name 3 - username
- **Stack Overflow**
  - Davide Bacci -[ Davide Bacci](https://stackoverflow.com/users/18345037/davide-bacci)
  - Name 2 - username
  - Name 3 - username
- **GitHub**
  - Name 2 -username
  - Name 2 - username
  - Name 3 - username
***
It would also be ideal to include something akin to Greg Deckler's [How To Get Your Question Answered Quickly](https://community.fabric.microsoft.com/t5/DAX-Commands-and-Tips/How-to-Get-Your-Question-Answered-Quickly/m-p/1626726#M32906) post, specifically tailored to Deneb/Vega-Lite/Vega.

INFO: @PBI-David

**Comment by dm-p** [comment]
Hi Madison - it's a great suggestion, and if people in the community want to put themselves out there explicitly to help others, I'm happy with having a section on this. [The doc site is also open source](https://github.com/deneb-viz/deneb-viz.github.io), and you are welcome to add the necessary sections or other content and submit a PR for review. If you would like some help with learning how to set up a dev environment and do this, I can go through it with you, but it will need to be after the 12th of January since I'm currently tied up with Global Power BI and Fabric Summit session prep and recording (and time is very tight!).

**Comment by Giammaria** [comment]
Thanks Daniel. Those plates never seem to stop spinning. 

I can take this for action and I think it'll be a good exercise. I've been wanting to become familiar with docusaurus anyway. 

 @PBI-David, let me know if you'd like me to add your SO account as well.

**Comment by dm-p** [comment]
As a heads-up, the README in that repo should be directionally correct, but I have since updated it to v3 of Docusaurus. It's not massively different in terms of how the site is architected or how you develop against it, but it might make a big difference as to what version of the doc you need to look at ;)

**Comment by PBI-David** [comment]
Yes, no problem @Giammaria. Thanks

**Comment by Giammaria** [comment]
This is what I have drafted currently. I figured it may be worth posting here for feedback/suggestions before doing a PR (let me know if a PR up front is preferred). I'm not married to any of this so feel free to make suggestions or raise any concerns.

Side note- @dm-p, don't feel like you have to provide immediate feedback. If it needs to wait until after the 12th, no worries. This is far from urgent.

Community & Resources Sidebar:
![image](https://github.com/deneb-viz/deneb/assets/16004891/bd9124b5-74ba-4304-8027-4809dbef178e)

Page
![image](https://github.com/deneb-viz/deneb/assets/16004891/89342437-1fa7-48bb-9e3c-cee4f23d7fbf)

INFO: @PBI-David

**Comment by PBI-David** [comment]
I would only add that they should search the hundreds of questions already solved to see if their particular question already has an answer. Stack Overflow probably has 90% of questions already covered if people searched for them properly. Also add that this place is not for Vega or VL questions but strictly for Deneb issues that they've encountered.

**Comment by Giammaria** [comment]
Great points all around. I'll update. Also, I agree, in hindsight "eager" comes off just a tad strong 😆

**Comment by dm-p** [comment]
@Giammaria, firstly, thank you very much for investing your time working on this. My initial thoughts are as follows:

- This may be easier to review in a PR, particularly if you have changes against the documentation repo you propose committing. You can assign reviewers, such as @PBI-David and myself, and it is easier to provide comments directly to parts of the markup that might need further tweaking, which is more contextual. As/when you make changes, we can repeat the cycle without worrying too much about keeping track of which comments apply to which context or version in an issue. Providing the changes as screenshots is excellent for those who might not be able to check out your branch and review the changes on a local dev environment also 👍

- We already have a [support page](https://deneb-viz.github.io/support) linked from the main toolbar, which, IMHO, is more discoverable. As we already highlight community resources to get free help on this page, might building it out with your content make more sense for people looking for help? Much of what you've written would significantly augment (or replace some of) the existing support page content for the better and keep things centralized. 

If chatting about these points is easier, I'm happy to set something up. I'll follow up with you both offline about this.

**Comment by Giammaria** [comment]
@dm-p, I replied offline, but reiterating support for your suggestions here.

Thank you for the detailed feedback. Those screenshots were taken from my dev environment so I am prepared to create a PR at any time. I have already made a few updates since those screenshots, including Davide’s suggestions. I also agree that leveraging the commenting capability that comes with PRs makes a lot of sense. 

If you’d like me to go ahead and create that PR prior to our discussion, just let me know. Otherwise if you prefer that I wait for the discussion, especially since we’ll want to likely update the more discoverable support page, then I will certainly hold off.
